cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -229,6 +244,38 @@ func detectEventHubsAccordingToSpringCloudEventhubsStarterDependency(
 	}
 }
 
+func detectEventHubsAccordingToSpringIntegrationEventhubsMavenDependency(
+	azdProject *Project, springBootProject *SpringBootProject) {
+	var targetGroupId = "com.azure.spring"
+	var targetArtifactId = "spring-cloud-azure-starter-integration-eventhubs"
+	if hasDependency(springBootProject, targetGroupId, targetArtifactId) {
+		newDep := AzureDepEventHubs{
+			// eventhubs name is empty here because no configured property
+			EventHubsNamePropertyMap: map[string]string{},
+			UseKafka:                 false,
+			MavenDependencyType:      SpringIntegrationEventHubs,
+		}
+		azdProject.AzureDeps = append(azdProject.AzureDeps, newDep)
```

Comment:
Continue the discusstion in this comment: https://github.com/azure-javaee/azure-dev/pull/85#discussion_r1895711131

Is it possible that `azdProject.AzureDeps = append(azdProject.AzureDeps, newDep)` executed multiple time? Will it work as expected when it executed multiple times?

Reply:
Same as above reply. Now it should work as expected when executed multiple times.